docs/en_us/configuration/server_data_conf/cluster_conf.data.md

```diff
@@ -6,69 +6,76 @@ cluster_conf.data records the cluster config.
 
 ## Configuration
 
-| Config Item | Description                                                   |
-| ----------- | ------------------------------------------------------------- |
-| Version     | String<br>Verson of config file                                         |
+| Config Item | Description                                                  |
+| ----------- | ------------------------------------------------------------ |
+| Version     | String<br>Version of config file                             |
 | Config      | Struct<br>Map data, key is cluster name, value is cluster config detail |
+| Config[k]   | String<br>Cluster name                                       |
+| Config[v]   | Object<br>Cluster's routing config                           |
 
 ### Cluster Config Detail
 
+Notice: below config are in namespace Config[v].
+
 #### Backend Config
 
 BackendConf is config for backend.
 
 | Config Item           | Description                                 |
 | --------------------- | ------------------------------------------- |
-| Protocol              | String<br>Protocol for conect backend, supported http and fcgi, default is http |
-| TimeoutConnSrv        | Int<br>Timeout for connect backend, in ms          |
-| TimeoutResponseHeader | Int<br>Timeout for read response header, in ms     |
-| MaxIdleConnsPerHost   | Int<br>Max idle conns to each backend              |
-| MaxConnsPerHost   | Int<br>Max number of concurrent conns to each backend              |
-| RetryLevel            | Int<br>Retry level if request fail                 |
-| BackendConf.OutlierDetectionHttpCode            | String<br> Http status code that represent error status of backend |
+| Protocol              | String<br>Protocol for connect backend, support http and fcgi. Default value is http |
+| TimeoutConnSrv        | Int<br>Timeout for connect backend, in ms. Default value is 2000 |
+| TimeoutResponseHeader | Int<br>Timeout for read response header, in ms. Default value is 60000 |
+| MaxIdleConnsPerHost   | Int<br>Max idle connections to each backend per BFE. Default value is 2. |
+| MaxConnsPerHost   | Int<br>Max number of concurrent connections to each backend per BFE. 0 means no limitation. Default value is 0. |
+| RetryLevel            | Int<br>Retry level if request fail. 0: retry after connect backend fails; 1: retry after connect backend fails or forwarding GET request fails. Default value is 0. |
```

Comment:
0: retry after connecting backend fails; 1: retry after connecting backend fails or forwarding GET request fails. Default value is 0.